apps/wallet-dashboard/components/dialogs/send-token/views/EnterValuesFormView.tsx

```diff
@@ -44,29 +45,30 @@ interface EnterValuesFormProps {
     onClose: () => void;
 }
 
+function totalBalance(coins: CoinStruct[]): bigint {
+    return coins.reduce((partialSum, c) => partialSum + getBalanceFromCoinStruct(c), BigInt(0));
+}
+function getBalanceFromCoinStruct(coin: CoinStruct): bigint {
+    return BigInt(coin.balance);
+}
+
 interface FormInputsProps {
     coinType: string;
     formattedTokenBalance: string;
     activeAddress: string;
     coins: CoinStruct[];
     isMaxActionDisabled: boolean;
     hasEnoughBalance: boolean;
+    setIsBuildingTransaction: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
I would rather move the state to the parent (I don't know why it was even put here in the first place 🤔 ) rather than synchronize local states through unnecessary abstraction 

Here is what I mean: https://github.com/iotaledger/iota/commit/5a427f5b3aa8a9940b90fe8783f0856983c75606